Smash_Combos.Core/Cqrs/Characters/DeleteCharacter/DeleteCharacterResponse.cs

```diff
@@ -4,7 +4,8 @@
 
 namespace Smash_Combos.Core.Cqrs.Characters.DeleteCharacter
 {
-    public class DeleteCharacterResponse : ResponseBase
+    public class DeleteCharacterResponse
     {
+        public bool Success { get; set; }
```

Comment:
Don't think you need these success parameters anymore, responses can remove these